### PR TITLE
fix: scrollview style flex

### PR DESCRIPTION
**Pull request by lybeen**
previous default style with flex 1 cause not working with style height, now fix the style problem with the alternative style flex

1. without flex style when style with height prop
2. with default flex style when style without height prop

### CODE REVIEW

File: packages/rax-scrollview/src/index.tsx [hunk 1]
```diff
@@ -244,6 +244,9 @@ const ScrollView: ForwardRefExoticComponent<ScrollViewProps> = forwardRef(
     const scrollerStyle: CSSProperties = {
       ...style
     };
+    if (scrollerStyle.height === undefined) {
+      scrollerStyle.flex = 1;
```
Comment (yacheng) [inline packages/rax-scrollview/src/index.tsx]: 此处的样式更改会影响现有业务的默认展现，建议升级一个大版本

File: packages/rax-scrollview/src/index.tsx [hunk 2]
```diff
@@ -244,6 +244,9 @@ const ScrollView: ForwardRefExoticComponent<ScrollViewProps> = forwardRef(
     const scrollerStyle: CSSProperties = {
       ...style
     };
+    if (scrollerStyle.height === undefined) {
```
Comment (yuanyan) [inline packages/rax-scrollview/src/index.tsx]: 需要考虑 === null 的情况

**Comment by yuanyan** [comment]
这应该是一个大版本升级，需要在 PR 升级下版本号

File: packages/rax-scrollview/src/index.tsx [hunk 3]
```diff
@@ -244,6 +244,9 @@ const ScrollView: ForwardRefExoticComponent<ScrollViewProps> = forwardRef(
     const scrollerStyle: CSSProperties = {
       ...style
     };
+    if (scrollerStyle.height === undefined || scrollerStyle.height === null) {
```
Comment (yuanyan) [inline packages/rax-scrollview/src/index.tsx]: 判断 == null 就可以了

**Pull request merged**